### ISSUE TITLE
`"gam"` (via mgcv?)

### ISSUE THREAD

**Issue by grantmcdermott** [grantmcdermott/tinyplot#239]

**Comment by vincentarelbundock** [comment]
Circling back to this, we already have splines, loess, and lm (which could be extended with polynomials). It's also trivial to add new types for users. Since this is an extra dependency and the added benefits are unclear, maybe we can close this?

Eventually, if people are interested, we could have a page on the website with user-written types.

**Comment by grantmcdermott** [comment]
Yeah, open to dropping. I'm heading out for a few days. We can discuss then!
